nav_msgs/OccupancyGrid map
float64[] data
